### ISSUE TITLE
Can you please add support to S8001-  iPad Pro (12.9-inch) 1st Gen

### ISSUE THREAD

**Issue by BrunoLegit** [MatthewPierson/PyBoot#53]
Hi Matthew 

Thank you for your fantastic project.

Is there any chance you can  add support to S8001 devices (iPad Pro (12.9-inch) 1st Gen)?

When I run the script I receive the following message:

"Please open an issue and let me know what device you are using/it's CPID and I will add support ASAP"

Do you need any extra information?

Can I do anything? Bear in mind I am not a dev. 

Many thanks

**Comment by shortbow123** [comment]
I don't believe the iPad Pro is even compatible with the checkm8 exploit, that might be why you are getting the error you are gettings. A11 and lower processors are supported, the 1st gen iPad Pro has an A12

**Comment by marijuanARM** [comment]
what? the first gen iPad Pro has an A9X processor, only the 3rd and 4th gen iPad Pros have an A12 Processor or higher

**Comment by shortbow123** [comment]
Sorry, I was looking at the wrong device, you might be able to add support yourself, you may want to poke around the code and see how other devices have gotten support, you can probably look up all the info you need online.
